### PR TITLE
Add symbol and file for softsprite_matrix_static

### PR DESCRIPTION
**Pull request by louist103**

### CODE REVIEW

File: include/variables.h [hunk 1]
```diff
@@ -4,7 +4,7 @@
 #include "z64.h"
 #include "segment_symbols.h"
 
-extern Mtx D_01000000;
+extern Mtx gSoftSpriteMartixStatic;
```
Comment (Dragorn421) [inline include/variables.h:7]:
```suggestion
extern Mtx gSoftSpriteMatrixStatic;
```
you had one job Sadge

File: src/code/softsprite_matrix_static.c [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#include "ultra64/types.h"
+
+//clang-format off
+Mtx gSoftSpriteMartixStatic = { 
+    65536 , 0     , 1     , 0     , 
+    0     , 65536 , 0     , 1     , 
+    0     , 0     , 0     , 0     , 
+    0     , 0     , 0     , 0     , 
+};
+//clang-format on
```
Comment (Dragorn421) [inline src/code/softsprite_matrix_static.c:10]: what does it look like without clang-format off/on?

(it doesn't make more sense to format it as a 4x4 grid, the first 8 values are the integer parts, the last 8 the fractional parts)

File: spec [hunk 3]
```diff
@@ -10466,5 +10466,6 @@ endseg
 beginseg
     name "softsprite_matrix_static"
     romalign 0x1000
-    include "build/baserom/softsprite_matrix_static.o"
+    include "build/src/code/softsprite_matrix_static.o"
```
Comment (Dragorn421) [inline spec:10469]: probably shouldn't be code but no clue where tbh
there are more of these so maybe a folder `src/segments/` ? no clue

**Review by Dragorn421 — CHANGES_REQUESTED** [review]

File: tools/ZAPD/ExporterTest/Jenkinsfile [hunk 4]
```diff
@@ -0,0 +1,97 @@
+pipeline {
+    agent {
+        label 'ZAPD'
+    }
+
+    stages {
+        // Non-parallel ZAPD stage
+        stage('Build ZAPD') {
```
Comment (AngheloAlf) [inline tools/ZAPD/ExporterTest/Jenkinsfile:8]: Looks like you messed up something here

File: spec (hunk 3 above)
Comment (AngheloAlf) [reply spec:10469]: Yeah, this definitely shouldn't be in `code/`. 
I like the  `segments/` idea

**Pull request closed without merging**